app/scripts/metamask-controller.js

```diff
@@ -1541,6 +1541,8 @@ export default class MetamaskController extends EventEmitter {
         if (!prevCompletedOnboarding && currCompletedOnboarding) {
           const { address } = this.accountsController.getSelectedAccount();
 
+          this._addAccountsWithBalance();
```

Comment:
We probably don't need to `await` this, lazyloading background accounts is probably fine.